### PR TITLE
Add example for applicationContext

### PR DESCRIPTION
**Pull request by masi**
test for subcontext

### CODE REVIEW

**Review by jonaseberle — APPROVED** [review]

**Comment by linawolf** [comment]
Should this be backported, @masi ?

**Comment by masi** [comment]
The example also applies to TYPO3 versions 9.5 and 10.x.

**Comment by mbrodala** [comment]
@linawolf It's unfortunate that my remark and change request has been ignored and we now have a broken example in our docs.

**Comment by linawolf** [comment]
@mbrodala I cannot see any previous remark or change request from you in this ticket? So this example is broken? Can you fix it and make another PR?

**Review by mbrodala — COMMENTED** [review]
@linawolf Dammit, I forgot to submit. Sorry for the accusation.

**Comment by mbrodala** [comment]
@linawolf I've just added #405 with the fix. Again, sorry for not noticing.

File: Documentation/Conditions/Index.rst [hunk 1]
```diff
@@ -49,6 +49,10 @@ applicationContext
 
       [applicationContext == "Development"]
 
+   Any context that is "Production" or starts with "Production" (eg Production/Staging"). ::
+
+      [applicationContext == "#^Production(/|$)#"]
```
[the remark this answers is not in the document]
Comment (jonaseberle) [reply Documentation/Conditions/Index.rst:54]: Oh you are so right. Sorry for my imprecise review!

[the remark this answers is not in the document]
Comment (masi) [reply Documentation/Conditions/Index.rst:54]: I always code for complete match, but forget that it is impossible to create a conetext named Productionitis so the strict check for "Production" is not needed.

[the remark this answers is not in the document]
Comment (masi) [reply Documentation/Conditions/Index.rst:54]: Sorry for the copy/paste error. I copied the line from the rst so the formatting is right and failed to replace the ==.

**Pull request merged**